qiskit/circuit/library/generalized_gates/mcmt.py

```diff
@@ -19,6 +19,8 @@
 from qiskit import circuit
 from qiskit.circuit import ControlledGate, Gate, Qubit, QuantumRegister, QuantumCircuit
 from qiskit.exceptions import QiskitError
+
+# pylint: disable=cyclic-import
 from ..standard_gates import XGate, YGate, ZGate, HGate, TGate, TdgGate, SGate, SdgGate
```

Comment:
This extra suppression is indicative to me that we've put a gate somewhere in the wrong place - I suspect it's `UnitaryGate`, which it might have been cleaner to give special treatment, since it's used by the base `Gate` class.  `standard_gates` shouldn't need to cyclically import `generalized_gates`.

We can re-arrange things in other PRs, though, no worries.

Reply:
This _might_ also be unnecessary -- after moving the gates there were loads of cyclic imports, and I might have added one too many ignores.... 🙈